### DISCUSSION TITLE
LOTUS Magazine

### DISCUSSION THREAD

**Discussion by taviso** [taviso/123elf#112] [General]
If you're interested in 1-2-3, you might enjoy LOTUS magazine - a monthly print magazine from the 80s and 90s all about 1-2-3!

It’s actually great, there are some really approachable articles on linear regression, forecasting, calculating bond yields, macro and formula tips, Q&A from readers, etc. The vast majority of the content is relevant to 1-2-3 UNIX!

![lotus-magazine](https://user-images.githubusercontent.com/123814/221380517-3b770995-44d1-4159-b6e8-634e5449e1fd.png)

There were 8 Volumes (12 issues per volume).

All issues are an archive.org, the early issues are only available from scanned microfiche, so they're in black and white, but later volumes are available in color.

Here are some examples, you can follow the links for more issues:

[Vol 3, Issue 1 (Jan 1987)](https://archive.org/details/sim_lotus_1987-01_3_1)
[Vol 4, Issue 1 (Jan 1988)](https://archive.org/details/sim_lotus_1988-01_4_1)
[Vol 5, Issue 1 (Jan 1989)](https://archive.org/details/sim_lotus_1989-01_5_1)
[Vol 6, Issue 1 (Jan 1990)](https://archive.org/details/sim_lotus_1990-01_6_1)
..
[Vol 8, Issue 1 (Jan 1992)](https://archive.org/details/lotus-magazine-v8i1)
etc, etc.

**Comment by rruhle** [comment]
Wow, nice find.
